toolkits/landcover/impl/composites.js

```diff
@@ -81,41 +81,45 @@ function createTemporalComposites(
 }
 
 /**
- * Computes a medioid composite by finding the median of one band
- * and returning the complete observation where that band is closest
- * to the median value.
+ * Computes a medioid composite by finding the median and returning the
+ * complete observation closest to the median value as measured by
+ * euclidiean distance over the specified band(s).
  *
  * @param {!ee.ImageCollection} collection The collection to composite.
- * @param {string|number} indexBand The name of the band which to
- *    select the median value. If omitted, the first band is used.
+ * @param {Array<string|number>|string|number} bands A band identifier
```

Comment:
If this arg is optional should be:
`@param {(Array<string|number>|string|number)=}`

Reply:
Done